ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf8 -*-
+"""
+Representation of general Kubernetes/OpenShift objects config file.
+
+This allows one to work with multiple objects of different kind at once, as
+explained in `Imperative Management of Kubernetes Objects Using Configuration
+Files
+<https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-config/>`_.
+
+Usage:
+    First you prepare list of dictionaries of k8s objects such as Deployment or
+    PVC which describes your workload/project to be deployed in OCP. Then
+    create instance of ``ObjectConfFile`` class with the list to be able to
+    create the resource in the cluster (to run it), or delete it later when
+    it's no longer needed.
+"""
+
+
+import logging
+import os
+
+import yaml
+
+from ocs_ci.utility.utils import run_cmd
+
+
+LOGGER = logging.getLogger(name=__file__)
```

Comment:
change 'LOGGER' with 'logger' or 'log' to keep consistency with other files

Reply:
Doing that will violate pep8, but I see that we are following this already and our ci doesn't complain, so ok.